ui/src/pages/ArtifactKey.vue

```diff
@@ -18,25 +18,24 @@
     ArtifactDescription,
     ArtifactTimeline,
     localization,
-    capitalize,
     useWorkspaceApi
   } from '@prefecthq/prefect-ui-library'
   import { useSubscription, useRouteParam } from '@prefecthq/vue-compositions'
   import { computed } from 'vue'
   import { usePageTitle } from '@/compositions/usePageTitle'
 
   const api = useWorkspaceApi()
-  const artifactId = useRouteParam('artifactId')
+  const artifactKey = useRouteParam('artifactKey')
 
-  const artifactSubscription = useSubscription(api.artifacts.getArtifact, [artifactId])
+  const artifactSubscription = useSubscription(api.artifacts.getArtifactCollection, [artifactKey])
   const artifact = computed(() => artifactSubscription.response)
```

Comment:
Adding a `useArtifact` composition is a nice pattern we've used in other places (`useFlow`, `useFlowRun`, etc) that you might want to adopt for artifacts.

Reply:
that composition exists, I created it and used it in exactly one place lol 🤦🏻. It won't be usable in this particular instance but there are other places it can be. I'll add a ticket to my backlog to update those and add a collection composition as well